manage_containers.py

```diff
@@ -104,27 +104,27 @@ def identify_runtime_cmd(self):
             sys.exit('[!] ERROR: the container runtime is not installed')
 
 def build_images(needed_compiler, images):
-    """Build container image(s) with the specified compiler"""
+    """Build container image(s) providing the specified compiler"""
     for c in images:
         if needed_compiler == 'all':
-            print(f'Adding Ubuntu-{c.ubuntu} container image with Clang-{c.clang} and GCC-{c.gcc}')
+            print(f'Add Ubuntu-{c.ubuntu} container image with Clang-{c.clang} and GCC-{c.gcc}')
```

Comment:
I think we should also rename Add to Build here and in other places, otherwise this blows the overall logic.